modern/src/runtime/System.ts

```diff
@@ -119,7 +119,10 @@ class System extends MakiObject {
   }
 
   getprivateint(section: string, item: string, defvalue: number): number {
-    if (!this._privateInt.has(section)) {
+    if (
+      !this._privateInt.has(section) ||
+      !this._privateInt.get(section).has(item)
```

Comment:
What if it’s zero?

Reply:
It's using `has`, I don't think the `section` can be zero?

Reply:
Oh totally. 👍